packages/bpk-storybook-utils/src/BpkStorybookUtils.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+/* eslint-disable global-require, no-console, import/no-mutable-exports */
+let action = (...args: Array<any>) => {
+  return () => console.log(args);
+};
+
+try {
+  const storybookAction = require('@storybook/addon-actions').action;
+  if (storybookAction) {
+    action = storybookAction;
+  }
+} catch (e) {
+  console.log(
```

Comment:
How about using `console.info` here instead? It'll make it easier to spot this message and it feels better semantically. 

<img width="326" alt="Screen Shot 2021-02-12 at 11 05 26" src="https://user-images.githubusercontent.com/73652/107760840-50645380-6d22-11eb-9a57-64efbf9ad619.png">


```suggestion
  console.info(
```